lldb/test/API/functionalities/memory/holes/TestMemoryHoles.py

```diff
@@ -0,0 +1,63 @@
+"""
+Test the memory commands operating on memory regions with holes
```

Comment:
I don't mind the terminology but perhaps add ` with holes (inaccessible pages)` to help anyone who is skimming through this. So they only have to read the header to know whether to care about the rest of the file.

Reply:
Done.